includes/Core/Admin/Screens.php

```diff
@@ -367,7 +367,10 @@ private function get_screens() {
 					'render_callback'  => function( Context $context ) {
 						$is_view_only = ! $this->authentication->is_authenticated();
 
-						$setup_slug = $context->input()->filter( INPUT_GET, 'slug', FILTER_SANITIZE_STRING );
+						$setup_slug = $context->input()->filter( INPUT_GET, 'slug' );
```

Comment:
@kuasha420 Is there a reason why we can't call `htmlspecialchars` here itself? 🤔

Reply:
The value of slug can be empty (null) and it throws another error for calling `htmlspecialchars` with `null` if we don't check.

Reply:
In that case @kuasha420 is it possible to use the Elvis operator here to be consistent with how it's done in other places where you used `htmlspecialchars`?